node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java

```diff
@@ -358,25 +358,6 @@ private void removeContainer(NodeAgentContext context, Container existingContain
         context.log(logger, "Container successfully removed, new containerState is " + containerState);
     }
 
-
-    private Container updateContainerIfNeeded(NodeAgentContext context, Container existingContainer) {
-        ContainerResources wantedContainerResources = getContainerResources(context);
-
-        if (healthChecker.isPresent() && firstSuccessfulHealthCheckInstant
-                .map(clock.instant().minus(warmUpDuration(context.zone()))::isBefore)
-                .orElse(true))
-            return existingContainer;
-
-        if (wantedContainerResources.equalsCpu(existingContainer.resources)) return existingContainer;
-        context.log(logger, "Container should be running with different CPU allocation, wanted: %s, current: %s",
-                wantedContainerResources.toStringCpu(), existingContainer.resources.toStringCpu());
-
-        // Only update CPU resources
-        containerOperations.updateContainer(context, wantedContainerResources.withMemoryBytes(existingContainer.resources.memoryBytes()));
```

Comment:
Shoot - this we actually want.  When a container starts, it is given no cap on the CPU until the warmup period is over to give the JVM sufficient resources to JIT the code, and in case the cap is reinstated.  This was implemented not too long ago and I'm guessing we want to continue supporting it, but we can look back to see whether the benefits still makes sense.  Orchestrating such a change defeats the purpose as the JVM is restarted.

@freva , did we discuss removing support for this too?

Didn't we conclude that implementing this would probably be as simple as maintaining the 2 corresponding cgroups values?  We should then also read them properly to ensure we're not getting stale data from podman.